gen/bbmri_collections

```diff
@@ -71,7 +73,8 @@ foreach my $resourceId ( $data->getResourceIds() ) {
 my @users;
 foreach my $uid (sort keys %$userStruc) {
 	my $user = {};
-	$user->{"id"} = $uid;
+	$user->{"id"} = "$userStruc->{$uid}->{$u_bbmri_user_id}";
+	$user->{"perun_user_id"} = $uid;
```

Comment:
Why do you need Perun user ID?

Reply:
Just wanted to let it there for debugging purposes, could be useful for further changes after services are re-connected to LS AAI proper proxy.